CMakeLists.txt

```diff
@@ -1,69 +1,62 @@
 cmake_minimum_required(VERSION 2.6)
+
 if(${CMAKE_VERSION} STREQUAL "2.8.2")
-  # bug http://vtk.org/Bug/view.php?id=11020
-  message( WARNING "CMake/CPack version 2.8.2 will not create working .deb packages!")
-endif(${CMAKE_VERSION} STREQUAL "2.8.2")
+	# Bug http://vtk.org/Bug/view.php?id=11020
+	message(WARNING "CMake/CPack version 2.8.2 will not create working .deb packages!")
+endif()
 
 # This can be read from ${PROJECT_NAME} after project() is called
-project(minetest)
+project(Minetest)
```

Comment:
It would be 100% simpler to just leave this lower-case and omit all the lowercase(project_name) code